publications.html

```diff
@@ -69,9 +69,9 @@ <h4 class="h4">Multi-Voltage Level Distributed Backward-Forward Sweep Power Flow
           <h4 class="h4">Agent-based Discrete-Event Simulation Environment for Electric Power Distribution System Analysis</h4>
           <div class="publication-meta-info"><img src="images/person.svg" loading="lazy" width="30" id="w-node-_28d94a92-1327-b6d4-eb06-bb2a116e9a09-ff010974" alt="" class="person-icon"><img src="images/calendar.svg" loading="lazy" id="w-node-_68627e07-eca0-8c0d-2443-2032f4efd0fb-ff010974" alt="" class="calendar-icon"><img src="images/location.svg" loading="lazy" id="w-node-_28d94a92-1327-b6d4-eb06-bb2a116e9a0b-ff010974" alt="" class="position-icon">
             <p id="w-node-_28d94a92-1327-b6d4-eb06-bb2a116e9a0c-ff010974" class="publication-meta-paragraph">Johannes Hiry</p>
-            <p id="w-node-_28d94a92-1327-b6d4-eb06-bb2a116e9a0e-ff010974" class="publication-meta-paragraph">publication in progress</p>
+            <p id="w-node-_28d94a92-1327-b6d4-eb06-bb2a116e9a0e-ff010974" class="publication-meta-paragraph">October 2021</p>
```

Comment:
```suggestion
            <p id="w-node-_28d94a92-1327-b6d4-eb06-bb2a116e9a0e-ff010974" class="publication-meta-paragraph">2021</p>
```

Because of the ambiguity of what actual date to use (day of handing in, day of defense, ...), I'd rather go with only having the year.